subsys/fs/Kconfig.littlefs

```diff
@@ -117,12 +117,19 @@ config FS_LITTLEFS_BLK_DEV
 	  Enable this option to provide support for littlefs on the block
 	  devices (like for example SD card).
 
-config FS_LITTLEFS_DISK_VERSION
+config FS_LITTLEFS_MULTIVERSION
```

Comment:
This can be done. You can not just rename the Kconfig and reuse the original name somewhere else, because that breaks existing configurations.

Reply:
Are you suggesting that I rename `FS_LITTLEFS_DISK_VERSION` to `FS_LITTLEFS_MULTIVERSION` but use a name (for example `FS_LITTLEFS_DISK_VERSION_NUMBER`) for the version number?

Or are you suggesting I don't rename at all, and directly use `FS_LITTLEFS_DISK_VERSION_NUMBER` for the version number?

Reply:
No renames. Just add new a Kconfig to control a new feature.